### PR TITLE
Us5 provide borrower names

### PR DESCRIPTION
**Pull request by danielmarker**
Borrower tests added - DO NOT MERGE UNTIL BORROWER CODE IS READY

### CODE REVIEW

File: features/functions/create_data.rb [hunk 1]
```diff
@@ -3,8 +3,41 @@ def generate_title_number
   title_number.generate
 end
 
-def create_deed_hash(title_number = generate_title_number)
```
Comment (Phemus) [inline features/functions/create_data.rb:6]: I noticed you've removed the title number overload is there a reason for this?

**Comment by Phemus** [comment]
Looks mostly good guessing the change from title to deed is to reflect the change in scope to the whole deed. see comment about overload being removed which was to allow for a specific title to be tested though

File: features/functions/create_data.rb (hunk 1 above)
Comment (danielmarker) [inline features/functions/create_data.rb:6]: It is replaced at the bottom - due a "basic deed" now needing the borrower parts also.

Comment (Phemus) [inline features/functions/create_data.rb:6]: fair enough in which case :+1:

**Comment by endamccormack** [comment]
:+1:

**Comment by danielmarker** [comment]
Overload test added - please additionally review

**Comment by tsangsimon** [comment]
:+1:  to the additional changes

**Pull request merged**